src/utils/registration-helpers.ts

```diff
@@ -809,7 +819,16 @@ export const convertToRegistrationSearchItem = (registration: Registration) => {
   const publisher =
     registration.entityDescription?.reference?.publicationContext &&
     'publisher' in registration.entityDescription.reference.publicationContext
-      ? registration.entityDescription.reference.publicationContext.publisher
+      ? {
+          id: registration.entityDescription.reference.publicationContext.publisher?.id,
+          name: registration.entityDescription.reference.publicationContext.publisher?.name,
+        }
+      : undefined;
+
+  const contextId =
```

Comment:
Kunne godt vært journalId som den var før for min del altså, siden det er det den egentlig er, men det jo ikke feil dette heller, så opp til deg! :D